### DISCUSSION TITLE
smart code comments in helix

### DISCUSSION THREAD

**Discussion by blitzerr** [helix-editor/helix#5412] [General] [answered]
When writing Rust block comments, I miss the feature where when you hit  the enter, it adds the comment marker like `///` or `//!`. How do people get around this ?

**Comment by the-mikedavis** [comment, marked as answer]
https://github.com/helix-editor/helix/pull/1937

**Comment by blitzerr** [comment]
Thanks @the-mikedavis  for the link. It looks like that issue has been stale for a while.

**Reply by the-mikedavis** [reply]
The author mentioned in the Matrix chat room that they were interested in working on it again soon.

I also have a more minimal commit that adds this in my fork which you can cherry pick if you're building from source: https://github.com/the-mikedavis/helix/commit/9ed9f3db264813554995e0cc1096e4f5450f9aef

**Reply by blitzerr** [reply]
Thanks a lot @the-mikedavis for pointing me to your fork. This is very helpful and I will try that.
